test/BUILD

```diff
@@ -286,6 +286,10 @@ pipeline_tests(
             "testdata/**/*.rb",
             "testdata/**/*.exp",
         ],
+        exclude = [
+            # Specific test that is modified for Prism, contains minor differences in the exp file
+            "testdata/rbs/assertions_heredoc_modified.rb",
```

Comment:
There are now 2 heredoc tests and they have minor differences in the `exp` file. I implemented a complex fix in https://github.com/sorbet/sorbet/commit/182e2f6fc8083e2b656928d8908fa9de180aaafe but we opted for keeping the implementation simpler and testing Prism differently instead.